next.config.js

```diff
@@ -1,4 +1,4 @@
 /** @type {import('next').NextConfig} */
 module.exports = {
-  reactStrictMode: true,
-}
+  // reactStrictMode: true,
```

Comment:
What does this do?

Reply:
I don't recall why I commented it out, I'll undo this change